hathor/util.py

```diff
@@ -759,3 +763,33 @@ def is_token_uid_valid(token_uid: bytes) -> bool:
         return True
     else:
         return False
+
+
+@dataclass
+class EnvironmentInfo:
+    # Changing these names could impact logging collectors that parse them
+    python_implementation: str
+    hathor_core_args: str
+    hathor_core_version: str
+    peer_id: Optional[str]
+    network: str
+
+    def as_dict(self):
+        return asdict(self)
+
+
+def get_environment_info(args: Namespace, peer_id: Optional[str]) -> EnvironmentInfo:
```

Comment:
Does argparse add parameters passed as envvars to the `Namespace` object?

Reply:
Yes, `Namespace` is a merge of the options from the cli and env after parsing them.

Reply:
Thanks!